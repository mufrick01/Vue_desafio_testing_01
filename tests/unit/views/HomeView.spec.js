import HomeView from "@/views/HomeView.vue"
import { shallowMount } from "@vue/test-utils"

describe('<HomeView/>',()=>{
    test('el componente debe coincidir con el snapshot',()=>{

        const wrapper = shallowMount(HomeView);

        expect(wrapper.html()).toMatchSnapshot();

    })
})