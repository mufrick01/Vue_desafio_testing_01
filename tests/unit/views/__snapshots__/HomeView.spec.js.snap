// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView/> el componente debe coincidir con el snapshot 1`] = `
<div class="home"><img alt="Vue logo" src="">
  <posts-view-stub></posts-view-stub>
</div>
`;
